Description: [PATCH] polkit: Ensure the cancellable is truly unowned
 Cancel button on polkit windows does not work.  This patch resolves this
Origin: Commit 48cf2594ea7d42a6492b7aaaea52e5bee9d3a8d0
Author: Ikey Doherty <ikey@solus-project.com>
Last-Update: 2018-05-01

---
 src/polkit/polkitdialog.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/polkit/polkitdialog.vala b/src/polkit/polkitdialog.vala
index 397579fc..5f677e07 100644
--- a/src/polkit/polkitdialog.vala
+++ b/src/polkit/polkitdialog.vala
@@ -68,7 +68,7 @@ public class AgentDialog : Gtk.Window
     }

     /* Manipulate Vala's pointer logic to prevent a copy */
-    public unowned GLib.Cancellable? cancellable { public set ; public get; }
+    public unowned GLib.Cancellable? cancellable { public set ; public unowned get; }

     public string cookie { public get; public set; }

--
2.17.1

